%YAML:1.0
use_loop_vins: 1
time_tolerance: 0.2
move_dis: 0 #0.005
imu_buffer_size: 200 #3ç§’
window_size: 5